libs/application/templates/health-insurance/src/lib/HealthInsuranceTemplate.ts

```diff
@@ -8,6 +8,11 @@ import {
 } from '@island.is/application/core'
 import * as z from 'zod'
 import { NO, YES } from '../constants'
+import {
+  isEUCountry,
+  requireConfirmationOfResidency,
+} from '../healthInsuranceUtils'
```

Comment:
These imports should have been removed in an earlier commit, perhaps they were added by accident in a merge conflict

Reply:
Done removed